### ISSUE TITLE
Build failure & warnings on Windows

### ISSUE THREAD

**Issue by SteveRussell33** [Simon-L/ModScript#2]
```
g++ -std=c++11 -Wsuggest-override  -Idep/include -fPIC -I../../include -I../../dep/include -MMD -MP -g -O3 -march=nehalem -funsafe-math-optimizations -fno-omit-frame-pointer -Wall -Wextra -Wno-unused-parameter -DARCH_WIN -D_USE_MATH_DEFINES -municode  -c -o build/src/LuneHelper.cpp.o src/LuneHelper.cpp
In file included from src/LuaJITEngineAPI.cpp:1:
src/Lune.hpp:5:10: fatal error: glob.h: No such file or directory
    5 | #include <glob.h>
      |          ^~~~~~~~
compilation terminated.
In file included from src/Lune.cpp:5:
src/Lune.hpp:5:10: fatal error: glob.h: No such file or directory
    5 | #include <glob.h>
      |          ^~~~~~~~
compilation terminated.
make: *** [../../compile.mk:72: build/src/Lune.cpp.o] Error 1
make: *** Waiting for unfinished jobs....
make: *** [../../compile.mk:72: build/src/LuaJITEngineAPI.cpp.o] Error 1
src/LuneHelper.cpp: In member function 'virtual void LuneHelper::process(const rack::engine::Module::ProcessArgs&)':
src/LuneHelper.cpp:77:86: warning: format '%ld' expects argument of type 'long int', but argument 3 has type 'size_t' {aka 'long long unsigned int'} [-Wformat=]
   77 |                                         dump += string::f("\t\t[\"%s\"] = {index = %ld},\n", lightName.c_str(), i);
      |                                                                                    ~~^                          ~
      |                                                                                      |                          |
      |                                                                                      long int                   size_t {aka long long unsigned int}
      |                                                                                    %lld
src/LuneHelper.cpp:79:75: warning: format '%ld' expects argument of type 'long int', but argument 2 has type 'size_t' {aka 'long long unsigned int'} [-Wformat=]
   79 |                                         dump += string::f("\t\t{index = %ld},\n", i);
      |                                                                         ~~^       ~
      |                                                                           |       |
      |                                                                           |       size_t {aka long long unsigned int}
      |                                                                           long int
      |                                                                         %lld
src/LuneHelper.cpp:89:45: warning: format '%lx' expects argument of type 'long unsigned int', but argument 3 has type 'int64_t' {aka 'long long int'} [-Wformat=]
   89 |                         sprintf(hexID, "0x%lx", hoveredModule);
      |                                           ~~^   ~~~~~~~~~~~~~
      |                                             |   |
      |                                             |   int64_t {aka long long int}
      |                                             long unsigned int
      |                                           %llx
src/LuneHelper.cpp: In member function 'virtual void LuneHelperWidget::step()':
src/LuneHelper.cpp:147:70: warning: format '%lx' expects argument of type 'long unsigned int', but argument 2 has type 'int64_t' {aka 'long long int'} [-Wformat=]
  147 |                                 moduleIdLabel->text = string::f("0x%lx", (int64_t)hoveredModule);
      |                                                                    ~~^   ~~~~~~~~~~~~~~~~~~~~~~
      |                                                                      |   |
      |                                                                      |   int64_t {aka long long int}
      |                                                                      long unsigned int
      |                                                                    %llx
src/LuneHelper.cpp:162:70: warning: format '%ld' expects argument of type 'long int', but argument 3 has type 'int64_t' {aka 'long long int'} [-Wformat=]
  162 |                                 paramLabel->text = string::f("%s (%2ld)", quant->getLabel().c_str(), (int64_t)hoveredParam);
      |                                                                   ~~~^                               ~~~~~~~~~~~~~~~~~~~~~
      |                                                                      |                               |
      |                                                                      long int                        int64_t {aka long long int}
      |                                                                   %2lld
src/LuneHelper.cpp: In constructor 'LuneHelperWidget::LuneHelperWidget(LuneHelper*)':
src/LuneHelper.cpp:181:21: warning: unused variable 'baseLightY' [-Wunused-variable]
  181 |                 int baseLightY = 72;
      |                     ^~~~~~~~~~
```

**Comment by Simon-L** [comment]
Hello and thank your for the report!

So far this plugin has only been tested on Linux but it is planned as one of the urgent next step to test and ensure it works on all platform. The Readme might be misleading, I will add this info! Today I'm banging my head against a wall trying to fix #1 

I guess you got here from the Rack forum? I expected there would be few Linux users, although I'm curious how diverse the OSes are, I'll ask about it in the thread!

Thanks

**Comment by SteveRussell33** [comment]
No problem, yeah read the thread on the forum and, like a few over there, am super interested in this project. I would say that Windows users are in the minority, but wouldn't know what the % split would be...

**Comment by Simon-L** [comment]
Fixed in [9b54df4](https://github.com/Simon-L/ModScript/commit/9b54df4e0e9a6572e3f07242d1ba832b4c8e2ab0)!

Built and tested on Windows 11.
Quick note: cross compile from Linux using the official docker plugin toolchain doesn't seem to work for this plugin (probably for VCV-Prototype too?), I had to compile directly on a Windows machine
